contributor-guide/modules/ROOT/pages/getting-involved.adoc

```diff
@@ -1,4 +1,60 @@
+
+* *Announcements*: The mailing list is a channel for updates, announcements about new releases, bug fixes, events, or any other relevant news.
+
+* *Collaboration*: The mailing list also serves as a platform for collaboration, fostering connections between developers and encouraging team-based projects.
+
+* *Discussion*: Developers discuss ideas, techniques, issues, and potential improvements related to Boost libraries.
+
+* *Support*: Newcomers or users encountering difficulties can seek help from the community. Experienced members provide solutions, advice, and share their experiences.
+
+If you do join, follow the https://www.boost.org/community/policy.html[guidelines and rules] of the mailing list, as it helps in maintaining a respectful and productive environment.
+
+To subscribe to the Developers Mailing List go https://lists.boost.org/mailman/listinfo.cgi/boost[Boost developers' mailing list].
+
+There are other mailing lists you might want to join, such as the https://lists.boost.org/mailman/listinfo.cgi/boost-users[Boost Users mailing list], or project-specific lists. Visit https://www.boost.org/community/groups.html[Boost Mailing Lists] for more details.
+
+== Report Issues and Suggest Improvements
+
+To formally report an issue on an individual library, locate the repo for that library. Many of these repos exist in the https://github.com/cppalliance/boost/tree/master/libs[Boost Super-project] on GitHub. Search under the *libs* folder. For example, to file an issue on the boost:asio[] library, click *New Issue* at https://github.com/boostorg/asio/issues[]. If you have difficulty locating the correct repo, then ask the question of the https://lists.boost.org/mailman/listinfo.cgi/boost[Boost developers' mailing list].
+
+
+To report an issue with the web site, including this documentation, click the *New Issue* button on this GitHub page: https://github.com/cppalliance/site-docs/issues[cppalliance/site-docs]. 
+
+After clicking *New Issue*, you will see a list of issue templates, such as *Submit a Correction or Bug report*, or *Suggest a Topic, Enhancement, or Information*. Select the most appropriate template (which will give you some advice on how to fill out the form), or select *Open a blank issue* if none of the templates applies.
+
+Whichever list you add your issue to, the issue will be addressed in due course. Providing full details, and examples and screenshots where appropriate, should give you the quicker response.
+
+== Contribute to an Existing Library
+
+Rather than report an issue, or suggested improvement, and wait for another developer to address it, you can make the code changes yourself. To be successful you will need to be familiar with GitHub, and obviously a competent pass:[C++] developer. After finding the issue, or coming up with the improvement, you will need to go to the GitHub repository for that library, and create a fork and clone the library.
```

Comment:
@alandefreitas do you think it would be worth to ask people to check whether the library has an active maintainer or not before they engage into this? To try to minimize wasted PRs

Reply:
Yes. That would be good. We could mention where to find the maintainers in the documentation, commit history, or `meta` files. Then they should probably check if the repository is very active. On the other hand, we don't want to make it sound like we have lots of unmaintained repositories. So I guess we should probably just mention how to find the maintainer at this point.

Reply:
Added the following paragraph:

Before filing an issue, it is good practice to identify the admins (authors or maintainers) of a library. This information can be found in the `meta` folder for the library. For example: https://github.com/boostorg/json/blob/develop/meta/libraries.json[] contains the authors for the JSON library example used above. Also, library documentation and commit history may well specify who is actively involved.